whynotearth/BrowTricks
# 396 add iOS app

# Issue Being Addressed

#396 

# Type of PR

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For New Features:** What feature are we adding? Explain your technical approach to doing this.

Add iOS app by Capacitor

# How to Test/Reproduce

`yarn install && yarn capacitor:build:production`, then if you have Xcode run `npx cap open ios`, see this video for the rest of work: https://youtu.be/y_UUjPkxlZ0?t=1034
